ocaml/gencert/selfcert.ml

```diff
@@ -107,22 +118,17 @@ let selfsign name alt_names length days certfile =
   in
   let privkey = `RSA rsa in
   let pubkey = `RSA (Rsa.pub_of_priv rsa) in
-  let issuer =
-    [X509.Distinguished_name.(Relative_distinguished_name.singleton (CN name))]
-  in
-  let req = X509.Signing_request.create issuer privkey in
```

Comment:
I'm not sure why this code was moved, it may make backporting more difficult

Reply:
I did it to reduce the number of params to the sign function, but you are right. I've moved the code back